.github/workflows/pr.yml

```diff
@@ -15,6 +15,11 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+      # We cannot use nancy-github-action because it is outdated, so it's better to use the latest
+      # docker image for the validation
+      - name: nancy
+        run: go list -m all | docker run -i sonatypecommunity/nancy:latest
```

Comment:
We might need to test it out for negative case to make sure it failed here. I faced a few cases github action didn't fail as expected, so just a precautious step.

Reply:
I'm using it in several projects, an at work and it's failing :)

Reply:
recently I had to disable it on a weekend because the server was unavailable

Reply:
oh I see, we can just let it run as non-required job for sometime first, then change settings later once we see fit.

Reply:
then I need to move it to a separate file

Reply:
like `extra`

Reply:
done

Reply:
oh actually, you can still have it in the same pr workflow, mandatory setting is per `job`. 

Anyway, no harm to keep it in another file.